Casks/b/bob-helper.rb

```diff
@@ -0,0 +1,18 @@
+cask "bob-helper" do
```

Comment:
```suggestion
cask "bobhelper" do
```

Please also update the filename to match.